packages/react/toast/src/Toast.tsx

```diff
@@ -233,20 +271,67 @@ const ToastViewport = React.forwardRef<ToastViewportElement, ToastViewportProps>
         tabIndex={-1}
         // incase list has size when empty (e.g. padding), we remove pointer events so
         // it doesn't prevent interactions with page elements that it overlays
-        style={{ pointerEvents: context.toastCount > 0 ? undefined : 'none' }}
+        style={{ pointerEvents: hasToasts ? undefined : 'none' }}
       >
+        {hasToasts && (
+          <FocusProxy
+            ref={headFocusProxyRef}
+            onEntranceFocus={() => focusFirst(getSortedCandidates({ direction: 'backwards' }))}
```

Comment:
On the way in these proxies focus via the sorted toast order, on the way out they implicitly "pass through" focus to the next valid element in the document, that latter is very helpful as we don't need to query for anything outside of our control. It'll even pass focus to browser controls at the very edges of the document.

Reply:
It's not super clear to me how the "way out" is handled?